skills/project-command.js

```diff
@@ -0,0 +1,20 @@
+module.exports = function(controller) {
+    // simple answer
+    controller.on('slash_command', function (bot, message) {
+        bot.reply(message, 'A link has been added');
+    });
+}
+
+// let exampleInput = {
+// channel_Name: 'int-project-bot',
+//  channel_id: 'c9367a',
+//  command: '/project',
+//  text: 'add google google.com'
+//  };
+//
+// let exampleInput2 = {
+//  channel_Name: 'int-project-bot',
+//  channel_id: 'c9367a',
+//  command: '/project',
+//  text: 'Add colby-portfolio colbyallen.co'
+// };
```

Comment:
Let's delete the comments from this file.

Reply:
Will do.